src/index.ts

```diff
@@ -192,13 +184,41 @@ export class ProjectConfig {
    * return a new ProjectConfig instance created with those options.
    */
   static loadConfigFromFile(filepath: string): ProjectConfig|null {
-    let configParsed = ProjectConfig.loadOptionsFromFile(filepath);
+    const configParsed = ProjectConfig.loadOptionsFromFile(filepath);
     if (!configParsed) {
       return null;
     }
     return new ProjectConfig(configParsed);
   }
 
+  /**
+   * Returns the given configJsonObject if it is a valid ProjectOptions object,
+   * otherwise throws an informative error message.
+   */
+  static validateOptions(configJsonObject: {}): ProjectOptions {
```

Comment:
Google style guide is to prefer {} over Object or any: https://github.com/google/ts-style/blob/master/tslint.json#L11

I don't recall the reasoning for `{}` over `Object`, but `any` is a type of last resort, because it will allow you to do anything to it. Here I want the opposite, I don't want it to be legal to do anything to configJsonObject except pass it into other methods that can accept anything.